src/main/resources/schemas/V60__smartdocuments_informatie_object_type.sql

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+ALTER TABLE ${schema}.smartdocuments_document_creation_template
+    ADD informatie_object_type_url VARCHAR NOT NULL;
```

Comment:
Ah, shame the URL is the unique ID for these things... thanks ZGW API..  Storing URLs in databases will always cause issues at some point.. Can't we strip the UUID from the URL and store that instead? And then use the env params to build up the URL? And don't we already do that for other data?

Reply:
Also, the DrcClient already uses UUIDs for EnkelvoudigInformatieObjecten right? So why the need for a URL here?

Reply:
Using UUID now